### PR TITLE
Feature/enable ci cd through GitHub actions

### PR DESCRIPTION
**Pull request by ThejashwiniSwamy**
### Your checklist for this pull request

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

**Comment by daniel88ferreira** [comment]
test

File: .github/workflows/github-actions-demo.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: GitHub Actions Demo
```
Comment (daniel88ferreira) [inline .github/workflows/github-actions-demo.yml]: Name of the pipeline and name of this file could be changed to reflect what this workflow actually does

**Review by daniel88ferreira — COMMENTED** [review]
Also there is some extra files in the same directory that can be deleted.

**Review by daniel88ferreira — COMMENTED** [review]
Please also remove all the unnecessary files from `.github` and `.github/workflows` folder

**Comment by ThejashwiniSwamy** [comment]
> Please also remove all the unnecessary files from `.github` and `.github/workflows` folder

**Comment by ThejashwiniSwamy** [comment]
> Please also remove all the unnecessary files from `.github` and `.github/workflows` folder

As the files as part of the official master branch, I would like to keep the github workflow files as is and just add the ones needed for use case.

**Pull request merged**